int8 goal_count
----
int8 cancelled
int8 reached